packages/flutter/test/widgets/slivers_test.dart

```diff
@@ -195,4 +195,17 @@ void main() {
     expect(find.text('B'), findsNothing);
     expect(find.text('BOTTOM'), findsOneWidget);
   });
+
+  testWidgets('Can override ErrorWidget.build', (WidgetTester tester) async {
+    const Text errorText = Text('error');
+    ErrorWidget.builder = (FlutterErrorDetails details) => errorText;
```

Comment:
@goderbauer - looking at this, I would have thought I'd have to reset this at the end of the test.  Are we missing a check for that, or does this just not need to be reset?

Reply:
Not resetting it is a problem - filed https://github.com/flutter/flutter/issues/30897